### PR TITLE
feat: initial sentry setup

### PR DESCRIPTION
**Pull request by tiftran**

### CODE REVIEW

**Comment by tiftran** [comment]
used what merino-py has as a reference

File: consvc_shepherd/settings.py [hunk 1]
```diff
@@ -140,3 +141,16 @@
         },
     },
 }
+# Sentry Setup
+SENTRY_SDK = env("SENTRY_SDK", default=None)
+SENTRY_TRACE_SAMPLE_RATE = env("SENTRY_TRACE_SAMPLE_RATE", default=1.0)
+sentry_sdk.init(
+    dsn=SENTRY_SDK,
+    integrations=[DjangoIntegration()],
```
Comment (ncloudioj) [inline consvc_shepherd/settings.py:149]: Need to specify the environment at the very minimum. 
```suggestion
    integrations=[DjangoIntegration()],
    environment=SENTRY_ENV,
```

File: consvc_shepherd/settings.py [hunk 2]
```diff
@@ -140,3 +141,16 @@
         },
     },
 }
+# Sentry Setup
+SENTRY_SDK = env("SENTRY_SDK", default=None)
```
Comment (ncloudioj) [inline consvc_shepherd/settings.py:145]:
```suggestion
SENTRY_DSN = env("SENTRY_DSN", default=None)
```

File: consvc_shepherd/settings.py [hunk 3]
```diff
@@ -140,3 +141,16 @@
...
+SENTRY_TRACE_SAMPLE_RATE = env("SENTRY_TRACE_SAMPLE_RATE", default=1.0)
+sentry_sdk.init(
+    dsn=SENTRY_SDK,
```
Comment (ncloudioj) [inline consvc_shepherd/settings.py:148]:
```suggestion
    dsn=SENTRY_DSN,
```

**Review by ncloudioj — APPROVED** [review]
r+ wc

**Pull request merged**